core/transfer/src/test/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImplTest.java

```diff
@@ -496,14 +496,15 @@ void verifyCompleted_notAllYetCompleted() throws InterruptedException {
             latch.countDown();
             return List.of(process);
         });
+        when(statusCheckerRegistry.resolve(anyString())).thenReturn((i, l) -> false);
```

Comment:
nit: just for readability, if you agree only.

```suggestion
        when(statusCheckerRegistry.resolve(anyString())).thenReturn((tp, resources) -> false);
```